### DISCUSSION TITLE
Implementing a way to share notes

### DISCUSSION THREAD

**Discussion by amitmerchant1990** [amitmerchant1990/notepad#19] [Ideas]
I'm thinking of a way to share notes across browsers, maybe through the URL itself.

But before that I'd need to check the browser limitations for that.

**Comment by AndreaLeah** [comment]
Are there any specific apps/sites that you'd like to be able to share to?

**Reply by amitmerchant1990** [reply]
Nothing specific. Just an ability to share the content with one click. But the problem we need to think about is what happens when the content is too big. How do we handle that?

**Reply by AndreaLeah** [reply]
I'm newer, so I'm not exactly sure on how to define how big, "too big" is. 

I suppose we could start with this:

1. Define how big is too big. 
2. Potentially use the values from the character count that the notepad functionality provides once #1 has been defined? (Use that count towards determining if the current text is inside or outside of the limit)
3. If the user tries to share something outside of #1's constraint, do we give an error message and display a message with the current character count and give the maximum character count value permitted for the share? Do we give them the ability to share selected portions at a time (characters 0-100)? Or would you like something else?
4. With the sharing functionality, what would you like the outcome to be? Only plain text shared? A specific format plus the text shared? (I'm not sure what exactly the vision for this is, or what style or way to go about it)
